core/src/main/java/com/capitalone/dashboard/repository/DashboardRepository.java

```diff
@@ -43,6 +43,8 @@
 	Iterable<Dashboard> findAllByConfigurationItemBusServNameAndConfigurationItemBusAppName(String appName, String compName);
 
 	Dashboard findByConfigurationItemBusServNameIgnoreCaseAndConfigurationItemBusAppNameIgnoreCase(String appName, String compName);
+	List<Dashboard> findAllByTypeContainingIgnoreCaseAndConfigurationItemBusServNameContainingIgnoreCase(DashboardType type, String appName);
+	List<Dashboard> findAllByTypeContainingIgnoreCaseAndConfigurationItemBusServNameContainingIgnoreCaseAndConfigurationItemBusAppNameContainingIgnoreCase(DashboardType type, String appName, String compName);
```

Comment:
Why do you need to specify type here? Only Team dashboards can have a BusApp/ BusSer. Can `findAllByConfigurationItemBusServNameAndConfigurationItemBusAppName` not be updated to add ignoreCase and used instead?

Reply:
When we create a production dashboard, there is an option on the UI to assign an Business Service, and therefore used type to make sure we are reading only Team dashboards

Reply:
You dont need "ignoreCase" wit type here.

Reply:
@tabladrum You are right, I will remove that. Thanks